snmp/datadog_checks/snmp/data/default_profiles/_dell-rac.yaml

```diff
@@ -6,6 +6,8 @@ metrics:
       OID: 1.3.6.1.4.1.674.10892.5.4.200.10
       name: systemStateTable
     symbols:
+      - name: systemState
+        constant_value_one: true
```

Comment:
I think we are in a similar situation than https://github.com/DataDog/integrations-core/pull/15471#discussion_r1284148892

Maybe it's also better to create a separate table.